target/linux/mediatek/mt7622/base-files/etc/init.d/bootcount

```diff
@@ -7,5 +7,13 @@ boot() {
 	linksys,e8450)
 		mtd erase senv || true
 		;;
+	xiaomi,redmi-router-ax6s)
+		local boot_wait=$( fw_printenv boot_wait | cut -d = -f 2 )
```

Comment:
Use `fw_printenv -n boot_wait` instead of pipe to `cut` for all three calls retrieving variables from uboot environment.